### ISSUE TITLE
The LPIPS Module Has a Normalize Parameter

### ISSUE THREAD

**Issue by Queuecumber** [Lightning-AI/metrics#1214]
## 🚀 Feature

I guess this is a feature request but the `lpips` module used by PM has a `normalize` parameter that allows users to pass tensors in the range [0, 1] instead of [-1. 1], PM should probably expose this parameter. The vast majority of pytorch codebases use [0, 1] for images.

### Motivation

I use image data in [0, 1] and I want to use lpips in a MetricsCollection, right now i need to either make all of the metrics work in [-1,1] and then renormalize my images or treat lpips as a special case.

### Pitch

I propose this be exposed in the constructor and set on every `forward` call to make the users life easier, it's unlikely that anyone is going to want to change it after it's set once.

So `init` becomes something like 

```
...
def __init__(
        self,
        net_type: str = "alex",
        normalize: False,
        reduction: Literal["sum", "mean"] = "mean",
        **kwargs: Any,
) -> None:
    ...
    self.normalize = normalize
```

and update becomes

```
def update(self, img1: Tensor, img2: Tensor) -> None:  # type: ignore
    ...
    loss = self.net(img1, img2, normalize=self.normalize).squeeze()
```
